client/app/bundles/course/assessment/submission/actions/attachments.js

```diff
@@ -1,20 +1,38 @@
-import attachmentsAPI from 'api/Attachments';
+import CourseAPI from 'api/course';
 
 import actionTypes from '../constants';
 
-export default function destroy(questionId, attachmentId) {
+export default function destroy(
```

Comment:
The deleteFile api is specific for text response, why is this destroy function placed under attachment.js?

Reply:
this attachment.js is actually under course/assessment/submission actions, so I firstly think it should not be a problem to leave it there (since it's already there, and it indeed does destroying the attachment of a submission, even though it's only for Text Response or File Upload question)

However, I re-think that the alternative of putting this function under index.js should be better (on one level with deleteProgrammingFile which is used for ProgrammingImportEditor). I have refactored it accordingly in my next push

Reply:
It's not fully correct also. the index.js file is supposed to be for the submission level. Does it make sense to create a separate files (i.e. text_response, programming) in a folder (answer) within actions? Also you can move the scribing file to the answer folder.